pkg/cmd/target/list.go

```diff
@@ -30,11 +30,17 @@ var targetListCmd = &cobra.Command{
 			return
 		}
 
-		if output.FormatFlag != "" {
-			output.Output = targets
+		if format.FormatFlag != "" {
+			// output.Output = targets
```

Comment:
Leftover

Reply:
done

Reply:
@joey1123455 in the future, please commit the review changes separately so it's easier for us to track and mark stuff as resolved.